### PR TITLE
Sample Sequence, Energy Scan UI, Bug fixes

### PR DESCRIPTION
**Pull request by marcus-oscarsson**
Hey,

Here is a big PR that contain six major categories of features/fixes, I should maybe have made several PRs instead. I thought it would be easier to handle like this, please let me know if you prefer that I divide
it. Each commit could almost be considered a PR on its own so you can treat as such.

The four major categories are
  - Improved sample by sample sequence
  - Bug fixes for bugs introduced when adding enable/disable sample
  - Optimized server side add_task_item
  - UI for Energy scan, including result handling on back-end
  - Various other minor bug fixes, like EDNA defaults
  - Some style fixes, like miss aligned elements

Solves issues #408, #407

Regards,
Marcus

### CODE REVIEW

File: config.gui.prod.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 module.exports = {
-	phaseControl: true,
+	phaseControl: false,
```
Comment (meguiraun) [inline config.gui.prod.js]: why did you remove this?

Comment (marcus-oscarsson) [reply config.gui.prod.js]: Yes I understand, but its false by default, you can enable it if you need it. I disabled it because most beamlines do not have phase control, so its like an additional option.
